.gitignore

```diff
@@ -104,6 +104,7 @@ celerybeat.pid
 # Environments
 .env
 .venv
+env
```

Comment:
Why the env?

Reply:
This is the file I used to update secrets in the bucket and didn't want to commit it accidentally.